src/AppInstallerCLICore/Workflows/ManifestComparator.h

```diff
@@ -62,10 +62,10 @@ namespace AppInstaller::CLI::Workflow
         };
     }
 
-    struct InstallerAndInapplicability
+    struct InstallerAndInapplicabilities
     {
         std::optional<Manifest::ManifestInstaller> installer;
-        InapplicabilityFlags inapplicabilityFlags;
+        std::vector<InapplicabilityFlags> inapplicabilitiesInstaller;

```

Comment:
It probably doesn't matter right now, but it seems odd to not know which installer each goes with.